### DISCUSSION TITLE
Reactor for Laravel

### DISCUSSION THREAD

**Discussion by lokmanm** [HassanZahirnia/laravel-package-ocean#104] [Package Suggestions]
Hey, first of all, great work on the Laravel Package Ocean!

I just wanted to recommend our recent package which is [Reactor for Laravel](https://github.com/lucky-media/reactor/tree/main). It's one of a kind starter kit for developers looking for a frontend-heavy setup with React, Inertia.js, and Laravel. 

Reactor is heavily influenced by [shadn/ui](https://ui.shadcn.com/) and its a port to Laravel.

We are actively maintaining it as we use it in our own projects.

**Comment by HassanZahirnia** [comment]
Thanks for your kind words Lokeman!

Also thanks for the suggestion 😊 I've added the package!
